packages/core-lib/src/index.ts

```diff
@@ -66,6 +68,18 @@ export const isNotEmpty = (value): boolean => {
     return !isEmpty(value); // String, Object
 };
 
+export const isObjectEqual = (objValue: object, othValue: object) => {
+    const _isEqual = isEqual(objValue, othValue);
+    if (_isEqual) return true;
+    const keys = union(Object.keys(objValue), Object.keys(othValue));
+    return isEqualWith(objValue, othValue, (a, b) => keys.every((key) => {
+        if (typeof a[key] === 'object' && typeof b[key] === 'object') {
+            return isObjectEqual(a[key], b[key]);
+        }
+        return isEqual(a[key], b[key]);
+    }));
```

Comment:
question: Can I think of it as 'deepEqual'?

Reply:
yes, like the isEqual.